golang/internal/dockerhelper/container.go

```diff
@@ -0,0 +1,109 @@
+package dockerhelper
```

Comment:
The naming is weird for me, using camelCase or snake_case would be better IMHO. The official documentation says these 2 can be okay for naming the packages. Link: https://go.dev/blog/package-names

I will vote for camelCase, but wait for @nandor-magyar and @robot9706 to agree on this.

Reply:
Name changed for a more hierarchical pattern, similar to containerbuilder.